docs/usage/tables/parquet.md

```diff
@@ -0,0 +1,120 @@
+
+## Set up Unity Catalog
+
+To follow along, make sure you have a local instance of Unity Catalog running by launching the following command from a terminal window:
+
+```sh
+bin/start-uc-server
+```
+
+This local UC server will come with some sample data pre-loaded.
+
+You can list all of the tables in your Unity Catalog using:
+
+```sh
+bin/uc table list --catalog unity --schema default
+```
+
+Your output should look something like this:
+
+```
+┌────────────┬────────────────┬────────────────┬────────────────┬────────────────┬────────────────┬────────────────┬────────────────┬────────────────┬────────────────┬────────────────┬────────────────┐
+│    NAME    │  CATALOG_NAME  │  SCHEMA_NAME   │   TABLE_TYPE   │DATA_SOURCE_FORM│    COLUMNS     │STORAGE_LOCATION│    COMMENT     │   PROPERTIES   │   CREATED_AT   │   UPDATED_AT   │    TABLE_ID    │
+│            │                │                │                │       AT       │                │                │                │                │                │                │                │
+├────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┤
+│user_coun...│unity           │default         │EXTERNAL        │DELTA           │[{"name":"fir...│file:///Users...│Partitioned t...│{}              │1721238005622   │1721238005622   │26ed93b5-9a18...│
+├────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┤
+│numbers     │unity           │default         │EXTERNAL        │DELTA           │[{"name":"as_...│file:///Users...│External table  │{"key1":"valu...│1721238005617   │1721238005617   │32025924-be53...│
+├────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┤
+│marksheet...│unity           │default         │EXTERNAL        │DELTA           │[{"name":"id"...│file:///tmp/m...│Uniform table   │{"key1":"valu...│1721238005611   │1721238005611   │9a73eb46-adf0...│
+├────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┤
+│marksheet   │unity           │default         │MANAGED         │DELTA           │[{"name":"id"...│file:///Users...│Managed table   │{"key1":"valu...│1721238005595   │1721238005595   │c389adfa-5c8f...│
+└────────────┴────────────────┴────────────────┴────────────────┴────────────────┴────────────────┴────────────────┴────────────────┴────────────────┴────────────────┴────────────────┴────────────────┘
+```
+
+As you can see, there are 4 tables in this catalog. All 4 pre-loaded tables are in the DELTA format.
```

Comment:
Suggest, re-writing this as "As you can see, there are currently four (4) Delta tables pre-loaded in this catalog.